azuread/resource_application.go

```diff
@@ -75,6 +76,16 @@ func resourceApplication() *schema.Resource {
 				Computed: true,
 			},
 
+			"group_membership_claims": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
I was asking this to not be case sensitive. Unless the API is returning varying values with random casing could we be case sensitive here?